README.md

```diff
@@ -1,9 +1,36 @@
-cat README.MD
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Nick Lesho . Jarvis Consulting
 
+I graduated with a Computer Science and Economics degree from the University of Toronto in June of 2021. Upon graduating, I worked as part of a 3-month contract for Canadian Musicians Co-operative as a Computer Systems Analyst. My duties included web development, IT support, and debugging an Android application. Currently, I am a technical consultant at Jarvis. I am passionate about web development, with proficiency in both front-end and back-end web development. I am especially passionate about using technical skills to solve problems for a wide variety of stakeholders. With the tech industry constantly changing and evolving, I am excited about learning and using all the new technologies that will unfold.
+
+## Skills
+
+**Proficient:** HTML, CSS, JavaScript, Python, Java, Linux/Bash, RDBMS / SQL, Agile / Scrum, Git
+
+**Competent:** React, Bootstrap, TypeScript, Django, REST API, Chrome Developer Tools, Wix
+
+**Familiar:** Angular.js, Vue.js, Spring Boot, Node.js, Express.js, MongoDB
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_NickLesho](https://github.com/jarviscanada/jarvis_data_eng_NickLesho)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_NickLesho/tree/master/linux_sql)]: Developed a program that provides information on hardware specifications, and real-time resource usage (such as CPU and memory) for each node within a system of nodes/servers connected through a network switch. The information is stored and retrieved from an RDBMS. Docker is used to provision the database and a psql instance is used to store all the data. A bash agent is used to gather server usage data and insert it into the database. The bash agent is composed of two bash scripts. The host_info.sh bash script inserts the host hardware info into the database and is run once. The host_usage.sh bash script collects the current host usage and inserts it into the database. It is run every minute using crontab. All of the source code is available on the project's GitHub repository.
```

Comment:
**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_NickLesho/tree/master/linux_sql)]: Developed a program that provides information on hardware specifications and real-time resource usage (such as CPU and memory) for each node within a system of nodes/servers connected through a network switch. The information is stored and retrieved from a PostgreSQL database which is provisioned using Docker. A Bash agent is used to gather server usage data and insert it into the database. Automate the bash agent using crontab. All of the source code is available on the project's GitHub repository.


Developed ABC for purposes using A, B, C. Bash script -> data collection. Persisted data in PostgreSQL. DA -> SQL. Crontab, testing..git

Reply:
https://github.com/jarviscanada/jarvis_data_eng_NarimanAlimuradov
https://github.com/jarviscanada/jarvis_data_eng_JasmeetBrar
https://github.com/jarviscanada/jarvis_data_eng_AlexKirilenko
https://github.com/jarviscanada/jarvis_data_eng_FatimaAnwer
https://github.com/jarviscanada/jarvis_data_eng_BrandonDicicco
https://github.com/jarviscanada/jarvis_data_eng_JohnCao
https://github.com/jarviscanada/jarvis_data_eng_TomasRotbauer
https://github.com/jarviscanada/jarvis_data_eng_TinaLee